lib/plotting_functions.py

```diff
@@ -1122,9 +601,9 @@ def plot_map_vect_and_save(wks, case_nickname, base_nickname,
     #Set stats: area_avg
     ax[0].set_title(f"Mean: {mdl_mag.weighted(wgt).mean().item():5.2f}\nMax: {mdl_mag.max():5.2f}\nMin: {mdl_mag.min():5.2f}", loc='right',
                        fontsize=tiFontSize)
-    ax[1].set_title(f"Mean: {obs_mag.weighted(wgt).mean().item():5.2f}\nMax: {obs_mag.max():5.2f}\nMin: {obs_mag.min():5.2f}", loc='right',
+    ax[1].set_title(f"Mean: {obs_mag.weighted(wgt).mean().item():5.2f}\nMax: {obs_mag.max():5.2f}\nMin: {mdl_mag.min():5.2f}", loc='right',
```

Comment:
Just double-checking that replacing `obs_mag` with `mdl_mag` is the correct thing to do here?  The same question holds for the `diff_mag` change below as well (and of course if I am just mis-understanding this change then please let me know!)